### PR TITLE
Add test for tables.remove method

### PR DESCRIPTION
**Pull request by sinchana-kumbale**
- [x] Wrote test for feature
- [ ] Added changes to CHANGELOG.md

**Changes proposed:**
Added a test for `tables.remove` in line with issue #476 
The test ensures method returns None and modifies no rows when called with an empty row list

### CODE REVIEW

File: tests/test_tables.py [hunk 1]
```diff
@@ -1158,6 +1158,11 @@ def test_remove_single(table):
     z      | 1     | 10
     """)
 
+def test_remove_none(table):
+	num_rows_before = table.num_rows
```
Comment (adnanhemani) [inline tests/test_tables.py:1162]: Seems like your tabs setting is set to 8 spaces versus the 4 spaces in the rest of this file. Can you please recheck this and make sure that the tabs look uniform?

**Review by adnanhemani — DISMISSED** [review]
This is great! Just one comment.

File: tests/test_tables.py (hunk 1 above)
Comment (sinchana-kumbale) [reply tests/test_tables.py:1162]: Thanks for the feedback on the tab spacing! I've made the changes to ensure consistency. Could you please take another look and confirm if it looks correct on your end?

Comment (adnanhemani) [reply tests/test_tables.py:1162]: Unfortunately, it doesn’t look like your new commit did anything :( the diff is empty and the indent still looks like 8 spaces (check the commits tab here on the PR). Can you please try again?

Comment (sinchana-kumbale) [reply tests/test_tables.py]: You were right, I did check and the commit had not changed the spacing. Please verify now. Thank you

Comment (davidwagner) [reply tests/test_tables.py]: Looks good to me, thank you!

**Review by davidwagner — APPROVED** [review]

**Pull request merged**